### PR TITLE
Ports Holoprojectors

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Axes police and engineering tape in favor of holoprojectors. Port from TG

Holo projectors are holographic signs that can be projected onto any turf.

How the projection behaves is dependent on what type of projection it is; engineering and police holo projections both block passage unless you're walking. Atmospherics holo projections, however, can be passed through; additionally, atmos projections block the passage of air, so they can act as a way to seal breaches temporarily.

Holo projections can be whacked with weapons, punched with your bare hands, or even shot, and they will dissipate; no more tape cheese blocking nukies off forever.

Changes:
- All tape is gone
- Engineers start with engineering projectors in their lockers
- Atmospheric technicians have atmos projectors in their lockers
- HoS, Warden, Detective, and Sec Officers have security projectors in their lockers
- Secborg has a sec projector
- Engiborg has an engineering projector

I'm leery of giving secborgs and regular officers these, but I'd be more than happy to have a discussion about it, as well

Pics:

img

Closes: https://github.com/ParadiseSS13/Paradise/issues/8973

:cl: Fox McCloud
del: Removes engineering and police tape
add: Adds security, atmospheric, and engineering holoprojectors
/:cl:

### CODE REVIEW

File: _maps/map_files/RandomZLevels/evil_santa.dmm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 "an" = (/obj/item/paper/journal_scrap_2,/turf/simulated/floor/carpet,/area/awaymission/challenge/main)
```
Comment (Fox-McCloud) [inline _maps/map_files/RandomZLevels/evil_santa.dmm]: Mapmerge refused to work with this, and out and out errored out.

**Comment by Fox-McCloud** [comment]
nothing like merge conflicts before you even start! Oh well!

**Comment by taukausanake** [comment]
I don't like the removal of tape. You can't keep both together and have one as an upgrade to the other? And make tape easier to remove/pass through if that's an issue?

**Comment by Fox-McCloud** [comment]
There's no point in having both, as they serve the same function.

**Comment by KasparoVy** [comment]
Welcome to the future my guy

**Comment by taukausanake** [comment]
I just like upgrade paths. Starting with something shit and working up to something better

**Comment by Spartan6** [comment]
I always liked tape though. Plus the atmos holo sounds and looks naff.

**Comment by Dinarzad** [comment]
If you are leary of giving these to borgs/sec officers the Projectors, then wouldn't keeping regular police tape for those positions be more prudent?
If they're not going to have access to their "Replacements" they need SOMETHING to cordon off a crime scene until the Detective/HoS can arrive.

**Comment by Ty-Omaha** [comment]
>There's no point in having both, as they serve the same function.

Mops and Space Cleaner also serve the same function

I like this but don't think tape should be removed.

**Comment by Tayyyyyyy** [comment]
I don't see what's wrong with this. It's a mostly a QOL improvement, and much more 26th century than tape. I do think sec officers should get these though.

**Comment by RyanSmake** [comment]
Give them to the respective borgs and sec officers, but regardless whether you do that, this seems fine! 👍

**Comment by davidchan13** [comment]
Not sure why borgs and officer wouldn't get these? If these are ported straight from tg anyone can just walk right through them, it just blocks running and ranged attacks.

How many holograms can each projector sustain?

**Comment by Spartan6** [comment]
Oh, sec and Borgs doesn’t get it?
Now I’m very against this.

**Comment by Fox-McCloud** [comment]
The atmospherics door needs to be graphically distinct from the others; it doesn't block movement like the others do, and functionally it exists to be a stop-gap for atmospheric problems. A door conveys this information way better than a free-standing barrier.

**Comment by tigercat2000** [comment]
@Fox-McCloud A big solid door looks like it'd block movement more than the barriers...

**Comment by tylerh95** [comment]
@Fox-McCloud why not give these to officers and secborgs? They both already have access to the tape.

**Comment by Vivalas** [comment]
I think tape should stay, since it's a nice aesthetic thing, like these. Really neither have a purpose.

**Comment by bigfatbananacyclops** [comment]
the holoprojectors haves a limit right?, id like tapes not to go too.

**Comment by Anticept** [comment]
You know what they need to look like?

Deployed forcefield emitters. Holo projectors just sounds weird, though they are in concept, similar.

I assume these will take a moment to deploy? Otherwise, these could be weaponized to show down pursuers like tables were.

**Comment by Fox-McCloud** [comment]
>I assume these will take a moment to deploy? Otherwise, these could be weaponized to show down pursuers like tables were.

Yep. They do. You can only deploy so many, per projector, as well.

**Comment by Spacemanspark** [comment]
I'd rather have both the tape and the holoprojector.

**Comment by Chopchop1614** [comment]
I actually like to use police tape as a security officer and this doesn't seem like a good substitute especially if you're trying to block a section of the corridor

**Comment by davidchan13** [comment]
@Chopchop1614 It does block movement, you have to manually slow down to walk to bypass a barrier to break it with attacks. 

Security barrier gets up to 6 active signs at once, so a 3 wide hallway could be blocked from two directions to prevent crime scene contamination, the secborg gets 9 active signs according to the code. If you need more than 6 tiles to block off a crime scene or block illicit access to an area, you need back up anyways.

**Comment by Fox-McCloud** [comment]
Secborg doesn't get 9.

I'm not buffing secborg by giving him the cyborg specific version which gives it even *more* power when emagged.

**Comment by Spartan6** [comment]
@davidchan13 It adds the item but does not give it to anyone.

**Pull request merged**